projects/module-2/whats-cookin-part-one.md

```diff
@@ -0,0 +1,277 @@
+
+You are expected to research and implement `beforeEach` in your test files.
+
+You are *expected* to test:
+* All functions that do not update the DOM.  This means everything in your index.js file.
+
+Remember to test all possible outcomes (happy/sad/etc).  Ask yourself:  
+  - Does the function return anything?  
+  - Are there different possible outcomes to test for based on different arguments being passed in?
+  <!-- - What do I expect if an argument is missing? -->
+
+You are *not expected* to test:
+
+* DOM manipulation / DOM manipulating function (like `document.querySelector(...)`)
+* Fetch calls
+
+---
+
+### Minimum Collaboration and Professionalism Expectations
+* Team holds daily standups throughout project.
+* Commits are atomic and frequent, effectively documenting the evolution/progression of the application. There is no more than a 10% disparity in project contributions between teammates.
+* A project board is utilized (and updated throughout the project) with Github issues and labels.
+* Team uses branches, PRs and thorough code reviews to add new code to the main branch.
+* The README is formatted well and at a minimum contains:
+  * Overview of project and goals
+  * Overview of technologies used, challenges, wins, and any other reflections
+  * Screenshots/gifs of your app
+  * List of contributors
+* **Team collaborates effectively to accomplish the shared goal.  Team productively and professionally works through challenges and conflicts to ensure all team members are able to be heard and contribute throughout the project.**  
+  * Instructors are available to offer support and guidance but conversations around what *is* and what *is not* working are expected to be led by the team members themselves.
+
+## Rubric
+
+For the rubric sections below, you will be scored as **Wow**, **Yes** or **Not Yet** depending on whether you have demonstrated competency in that area. Each section lists examples of what types of things we may be looking for as demonstrations of competency. Just as there are many ways to approach code, there are many many ways to demonstate competency.  There are just some examples. 
+
+### Functional Expectations
+* Wow: Application fulfills all requirements *as well as* an extension.
+* Yes: Application fulfills all requirements of iterations 1-3 without bugs.
+* Not Yet: Application crashes or has missing functionality or bugs.
+
+### Fundamental JavaScript & Style / OOP / Fetch
+- Code is divided into logical components each with a clean, single responsibility
+- Array prototype methods are used to iterate instead of for loops
+- All DOM manipulation is held in the `domUpdates.js`.  No DOM manipulation outside of this file.
+- Variables and functions are consistently and appropriately named
+- Code leverages JavaScript's truthy/falsey principles
+<!-- purity?  closures? -->
```

Comment:
Yes, I think that including something about purity is great!

Reply:
Added